File: src/main/java/com/hank/Hello.java
package com.hank;

public class Hello {
    public static void main(String[] args) {
//        System.out.println("Hello world");
        Person p = new Person("strong",1.7f,66.5f);
        p.hello();
        /*p.name="Strong";
        p.height=1.7f;
        p.weight=66.5f;*/
        System.out.println(p.bmi());
        /*int age = 19;
        Integer age2 = 19;*/
    }
}

File: src/main/kotlin/com/hank/Hello.kt
package com.hank

fun main() {
//    println("Hello World")
//    Human().hello()
    val h = Human(weight = 66.5f,height = 1.7f)
    h.hello()
    println(h.bmi())
   /* val a = 19
    var name : String
    name ="strong"*/
}

class Human(var name:String="",var weight:Float,var height:Float){
    init { //額外程式 建構子完成執行
        println("test $weight")
    }
//    constructor(name:String,weight: Float,height: Float):this(weight,height)
    fun bmi():Float{
        val bmi=weight/(height*height)
        return bmi
    }
    fun hello(){
        println("Hello kotlin")
    }
}